packages/payment-integration-test-framework/src/fixture/pageObject/playwright/ServerSideRender.ts

```diff
@@ -4,7 +4,7 @@ import ejs from 'ejs';
  * @internal
  */
 export class ServerSideRender {
-    async renderFile(filePath: string, data?: {}): Promise<string> {
+    async renderFile(filePath: string, data?: any): Promise<string> {
```

Comment:
Is it possible to type this `data` property?

Reply:
How about `Record<string, unknown>`?

Reply:
Seems good to me, and then you can use typegurad to validate it.